pytest_fixtures/component/provisioning_template.py

```diff
@@ -91,7 +91,15 @@ def module_sync_kickstart_content(
         )
         task_status = module_target_sat.api.ForemanTask(id=task['id']).poll()
         assert task_status['result'] == 'success'
-    rhel_xy = Version(constants.REPOS['kickstart'][repo_name]['version'])
+
+    rhel_xy = Version(
+        constants.REPOS['kickstart'][f'rhel{rhel_ver}_bos_beta']['version']
+        if rhel_ver == 10  # TODO: Remove beta repos once RHEL10 is GA
+        else constants.REPOS['kickstart'][f'rhel{rhel_ver}']['version']
+        if rhel_ver == 7
+        else constants.REPOS['kickstart'][f'rhel{rhel_ver}_bos']['version']
+    )
```

Comment:
In cases like these, normal if/elif/else blocks are the same number of lines and flow better.
```suggestion
    if rhel_ver == 10  # TODO: Remove beta repos once RHEL10 is GA
        rhel_xy = Version(constants.REPOS['kickstart'][f'rhel{rhel_ver}_bos_beta']['version'])
    elif rhel_ver == 7:
        rhel_xy = Version(constants.REPOS['kickstart'][f'rhel{rhel_ver}']['version'])
    else:
        rhel_xy = Version(constants.REPOS['kickstart'][f'rhel{rhel_ver}_bos']['version'])
```